index.html

```diff
@@ -1890,34 +1960,9 @@ <h2>
 
       <p>
 [[!JSON-LD]] is a JSON-based format used to serialize
-<a href="http://www.w3.org/TR/ld-glossary/#linked-data">Linked Data</a>. The
```

Comment:
I disagree with this approach, since all the text is available in document history and it makes no sense outside of its original section. Still, I have added it back in to a new appendix as requested.

Reply:
Ok, good, thanks. 

Also, please don't resolve conversations that you didn't start... the original commenter (me) doesn't see them, so it just looks like you closed the comment w/ no further consideration... and I have to click "unhide conversation" to see what you said.